### DISCUSSION TITLE
MSI RTX 2070 Ventus or MSI B450 A-PRO MAX?

### DISCUSSION THREAD

**Discussion by prettyleaf** [terminatorul/NvStrapsReBar#51] [General]
Anyone tested driver with one of these? Would appreciate the feedback

**Comment by Mommy86** [comment]
.codex-plugin/
└── plugin.json   — Codex plugin manifest (name, version, skills ref, MCP ref)
.mcp.json         — MCP server configurations at plugin root (NOT inside .codex-plugin/)
